packages/fxa-content-server/app/scripts/lib/channels/receivers/web-channel.js

```diff
@@ -88,7 +87,7 @@ _.extend(WebChannelReceiver.prototype, Backbone.Events, {
    */
   _reportError(error) {
     this._logger.error('WebChannel error:', error.message);
-    Raven.captureMessage('WebChannel error: ' + error.message, {
+    Sentry.captureMessage('WebChannel error: ' + error.message, {
```

Comment:
It'd be great to know why this wasn't caught as a full stack trace, but I understand this is mainly an update.